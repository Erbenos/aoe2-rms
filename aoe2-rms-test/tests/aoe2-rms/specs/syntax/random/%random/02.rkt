#lang aoe2-rms
; Have a 10% chance of placing 5 gold mines (and a 90% of not doing so)

<OBJECTS-GENERATION>
(%random [10 (create-object 'GOLD
                            (number-of-objects 5))])
