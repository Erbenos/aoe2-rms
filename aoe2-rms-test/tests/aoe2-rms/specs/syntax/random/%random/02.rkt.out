<OBJECTS_GENERATION>
start_random
  percent_chance 10
    create_object GOLD {
      number_of_objects 5
    }
end_random
